game/graphics/pipelines/opengl.cpp

```diff
@@ -209,6 +214,11 @@ static std::shared_ptr<GfxDisplay> gl_make_display(int width,
   }
 
   auto display = std::make_shared<GLDisplay>(window, is_main);
+
+  display->set_imgui_visible(Gfx::get_debug_menu_visible_on_startup());
+  display->update_cursor_visibility(window, display->is_imgui_visible());
+  glfwSetInputMode(window, GLFW_STICKY_KEYS, GLFW_TRUE);
```

Comment:
Is this needed? This turned out to be the cause of the weird double-alt-callback on linux.
From https://www.glfw.org/docs/3.3/input_guide.html it sounds like this only changes stuff with `glfwGetKey`, which we don't use.

Reply:
Gotcha, I'll remove it